.github/workflows/tests.yml

```diff
@@ -0,0 +1,61 @@
+name: Run tests
+
+on:
+  push:
+    branches:
+      - 'test'
+  pull_request:
+    branches:
+      - 'main'
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.11
+
+      - name: Set secrets in .env file
+        run: |
+          cd ./src
+          echo "SLACK_SIGNING_SECRET=${{ secrets.SLACKSIGNINGSECRET }}" >> .env
+          echo "SLACK_OAUTH_TOKEN=${{ secrets.SLACKOAUTHTOKEN }}" >> .env
+          echo "SLACK_VERIFICATION_TOKEN=${{ secrets.SLACKVERIFICATIONTOKEN }}" >> .env
+          echo "OPENAI_API_KEY=${{ secrets.OPENAIKEY }}" >> .env
+          echo "PINECONE_INDEX=${{ secrets.PINCONEID }}" >> .env
```

Comment:
I can find `PINECONE_INDEX` in the code but can't find `secrets.PINECONEID` defined anywhere. I suggest making them the same name, `PINECONE_INDEX`. (Assuming they represent the same value.)